lib/cocoapods-core/cdn_source.rb

```diff
@@ -195,20 +192,26 @@ def search_by_name(query, full_text_search = false)
       end
     end
 
-    # Does nothing, since CDN-backed repo is updated live.
+    # Check update dates for all existing files.
+    # Does not download non-existing specs, since CDN-backed repo is updated live.
     #
     # @param  [Bool] show_output
     #
-    # @return  [Array<String>] changed_spec_paths
-    #          Returns the list of changed spec paths.
+    # @return  [Array<String>] Always returns empty array, as it cannot know
+    #          everything that actually changed.
     #
     def update(_show_output)
-      debug 'No need to update CDN-backed repo'
+      @check_existing_files_for_update = true
+      begin
+        preheat_existing_files
+      ensure
+        @check_existing_files_for_update = false
+      end
       []
     end
 
     def git?
-      false
+      true
```

Comment:
is this change correct?

Reply:
Well, yes and no. 🙃

Yes, because the CDN source this is modeled on is a mirror of a git repo.

No, because it's not technically a git repo.

Personally, according to all the uses, I think the property should be called `can_be_updated?` on the base class, but this will require another PR in main.

Reply:
Probably should be broken out into 2 different atrtibutes, `git?` and `updatable?`, and the default implementation for `updatable?` can return `git?`

Reply:
Hehe... However from what I've seen there are no other uses of `git?`, so YAGNI 🤷🏻‍♂️

Reply:
It is a public method though, and I'm worried about it being a breaking change (potentially) for other source plugins

Reply:
>  Yes, because the CDN source this is modeled on is a mirror of a git repo. No, because it's not technically a git repo.

Worthwhile for a code comment actually.